capellambse_context_diagrams/collectors/exchanges.py

```diff
@@ -259,6 +261,9 @@ def add_interface(self) -> None:
             is_hierarchical=False,
         )
         src, tgt = generic.exchange_data_collector(ex_data)
+        self.data.edges[-1].layoutOptions = copy.deepcopy(
```

Comment:
Why is deepcopy necessary here or why don't do this with any other layout options e.g. makers.DEFAULT_LABEL_LAYOUT_OPTIONS?

Reply:
A copy is definitely to be used here... I am using deepcopy because mypy doesn't accept `.copy` because of our typing.